### PR TITLE
Restful api

### PR DESCRIPTION
**Pull request by DenisLukayenka**

### CODE REVIEW

**Review by Siarhei-Sialitski — COMMENTED** [review]
Are you going to implement few of the endpoints that will follow the 3rd level of maturity model?

File: src/Cart.Service/Cart.Service/Endpoints/CartEndpoints.cs [hunk 1]
```diff
@@ -9,68 +9,67 @@ public static class CartEndpoints
 
     public static WebApplication RegisterCartEndpoints(this WebApplication app)
     {
-        var cartApi = app.MapGroup("/api");
+        var versionOne = new Asp.Versioning.ApiVersion(1, 0);
+        var currentVersion = new Asp.Versioning.ApiVersion(Program.CurrentVersion.Major, Program.CurrentVersion.Minor);
```
Comment (Siarhei-Sialitski) [inline src/Cart.Service/Cart.Service/Endpoints/CartEndpoints.cs]: This is not covered with requirements, but I would not suggest couple api versioning and assembly version.
In your scenario when you want add endpoints with 3 rd version, you will need to update all routes for 2nd veriosn from 'currnetVersion' to versionTwo

Comment (DenisLukayenka) [reply src/Cart.Service/Cart.Service/Endpoints/CartEndpoints.cs]: Good point, updated

**Comment by DenisLukayenka** [comment]
> Are you going to implement few of the endpoints that will follow the 3rd level of maturity model?

Updated GET endpoint for Cart service

File: src/Cart.Service/Cart.Service/Mappings/CartMapping.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    public static ModelResponse<Models.Cart> MapToResponse(this Models.Cart cart)
+    {
+        var response = new ModelResponse<Models.Cart>()
+        {
+            Payload = cart,
+            Actions = [
+                new (){
+                    Method = HttpMethods.Delete,
+                    Rel = HttpMethods.Delete,
+                    Uri = $"/carts/{cart.Id}"
```
Comment (DenisLukayenka) [inline src/Cart.Service/Cart.Service/Mappings/CartMapping.cs:16]: Unfortunately, AOT does not work with `LinkGenerator` and `RouteValueDictionary`, so i have to use hardcode

**Review by Siarhei-Sialitski — APPROVED** [review]
LGTM!

**Pull request merged**